packages/lab/src/pagination/GoToInput.tsx

```diff
@@ -104,31 +101,26 @@ export const GoToInput = forwardRef<HTMLSpanElement, GoToInputProps>(
           className
         )}
         ref={forkedRef}
+        {...restProps}
       >
-        <FormField
-          className={withBaseName("goToInputField")}
-          fullWidth={false}
-          label={label}
-          labelPlacement="left"
-          {...restFormFieldLegacyProps}
-        >
-          <Input
-            className={clsx(withBaseName("goToInput"), {
-              [withBaseName("goToInputFixed")]: count < 100,
-            })}
-            id={id}
-            inputProps={{
-              "aria-labelledby": id,
-              "aria-label": `Page, ${count} total`,
-              style: { width: widthCh },
-            }}
-            onBlur={onBlur}
-            onChange={onChange}
-            onKeyDown={onKeyDown}
-            value={inputValue}
-            textAlign={"center"}
-          />
-        </FormField>
+        {label && <FormFieldLabel id={labelId}>{label}</FormFieldLabel>}
```

Comment:
Is there a reason why we aren't using FF here?

Reply:
It's because of the styling. Label is part of a gris template and will display on two rows

Reply:
Even with left label placement?

Reply:
especially with the left label placement

Reply:
I've added FF back and overriding styles